ceph.spec.in

```diff
@@ -536,6 +577,21 @@ librbd1 or libcephfs1 instead.
 
 %endif
 
+%package compat
```

Comment:
@ktdreyer should I also remove ceph-devel-compat, python-ceph-compat, and ceph-libs-compat? According to my audit those are empty as well (http://pad.ceph.com/p/package_contents)